_testcafe_tests/player_spec.ts

```diff
@@ -0,0 +1,130 @@
+/*
+    Player Page test suite - 2018 (c)
+ */
+
+import {Selector,ClientFunction} from 'testcafe';
+import selectors from '../src/selectors.js'
+import {tcUtils} from '../src/tc_utils'
+const testconfig = require('./testconfig');
+const texts = require('../src/texts');
+const player_utils = require('../src/player_utils');
+
+const width = 1400;
+const height = 1080;
+
+const setHeight = ClientFunction((selector) => {
+    const el = document.querySelector(selector);
+
+    el.style.width = '1px';
+    el.style.height = '1px';
+});
+
+
+fixture`Video Player Test Suite`
+    .page(testconfig.resources.playerUrl);
+
```

Comment:
this will not work 
const link = `${config.basePath}/${config.lang}/lessons`;
fixture`Lectures & Lessons`.page(`${link}`);